### ISSUE TITLE
Four-sided keyboard focus indicator is not visible on audio.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5740]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When the four‑sided keyboard focus indicator is not visible on audio controls, users who rely on keyboard navigation cannot determine which element currently has focus. This makes it difficult or impossible to operate audio playback, pause, or volume controls without a mouse.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Audio'.
3. Navigate to Pay/pause control present in bot response.
4. Observe the issue that Four-sided keyboard focus indicator is not visible on audio. 

### What do you expect?

Four-sided keyboard focus indicator should be visible on audio. 

### What actually happened?

Four-sided keyboard focus indicator is not visible on audio. 

### Do you have any screenshots or recordings to repro the issue?

<img width="1920" height="1020" alt="Image" src="https://github.com/user-attachments/assets/68f34e70-ca19-4a01-b977-2d516d040efd" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

### Adaptive Card JSON

```json

```
